SEP-0008.md

```diff
@@ -0,0 +1,179 @@
+    incremented if any public API functionality is marked as deprecated. It MAY
+    be incremented if substantial new functionality or improvements are
+    introduced within the private code. It MAY include patch level changes.
+    Patch version MUST be reset to 0 when minor version is incremented.
+
+1.  Major version X (X.y.z | X > 0) MUST be incremented if any backwards
+    incompatible changes are introduced to the public API. It MAY include minor
+    and patch level changes. Patch and minor version MUST be reset to 0 when
+    major version is incremented.
+    
+1.  A pre-release version MAY be denoted by appending 'a', 'b' or 'rc' followed
+    by an integer number. The 'a' identifier indicates an alpha release, 'b' a
+    beta release and 'rc' a release candidate release. These pre-release
+    identifiers are ordered such that a < b < rc, with higher numerical values
+    used to determine precedence between versions of the same pre-release type.
+    For example 1.0.0b100 < 1.0.0rc1. Pre-release versions have a lower
+    precedence than the associated normal version. A pre-release version
+    indicates that the version is unstable and might not satisfy the intended
+    compatibility requirements as denoted by its associated normal version.
+    Examples: 0.9.0a1, 1.2.3rc1, 2.0.0b10
+    
+1.  Development release numbers can be used to specify versions taken from
+    version control or not publicly released. These versions MUST NOT be
+    uploaded to indexes such as PyPI or anaconda.org. A development release MAY
+    be denoted using .devM where M is a non-negative integer number. Development
+    releases are ordered by their numerical component, immediately before the
+    corresponding release (and before any pre-releases with the same release
+    segment), and following any previous release (including any post-releases).
+    Development releases MAY also be used in conjunction with pre-release
+    numbering.
+    Examples: 1.0.0.dev25, 1.2.3a1.dev4532 1.0.0rc2.dev3456
+
+1.  Precedence refers to how versions are compared to each other when
+    ordered. Precedence MUST be calculated by separating the version
+    into major, minor, patch and pre-release identifiers in that order.
+    Precedence is
+    determined by the first difference when comparing each of these
+    identifiers from left to right as follows: Major, minor, and patch
+    versions are always compared numerically. Example: 1.0.0 < 2.0.0
+    < 2.1.0 < 2.1.1. When major, minor, and patch are equal, a
+    pre-release version has lower precedence than a normal version. Example:
+    1.0.0a1 < 1.0.0. Precedence for two pre-release versions with the
+    same major, minor, and patch version MUST be determined by following the
+    precedence rules for pre-release versioning. PEP 440 describes a formal way
+    of matching version numbers, which this specification is compatible with.
+    
+
+# Future SunPy Releases
```

Comment:
This section mixes plans for SunPy's development and the semantic versioning.  I recommend that this section be dropped.

Reply:
I agree.